internal/signing/azure_keyvault.go

```diff
@@ -227,3 +243,45 @@ func convert1363ToAsn1(b []byte) ([]byte, error) {
 
 	return asn1.Marshal(rs)
 }
+
+func GetKeyvaultAuthorizer() (autorest.Authorizer, error) {
```

Comment:
From reading the SDK, I was under the impression that this was handled automatically by NewAuthorizerFromEnvironment and you can set environment variables.